docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -38,18 +38,23 @@ details, refer to <<fleet-server-deployment>>.
 
 [discrete]
 [[fleet-server-compatibility]]
-== Compatibility
+== Compatibility and prerequisites
 
 {fleet-server} is compatible with the following Elastic products:
 
-* {stack} 7.13 or later (our {ess-product}[hosted {ess}] on {ecloud} or
+* {stack} 7.13 or later ({ess-product}[hosted {ess}] on {ecloud}, or
 a self-managed cluster).
+** For version compatibility: {es} >= {fleet-server} >= {agent} (except for
+bugfix releases)
+** {kib} should be on the same minor version as {es}.
 
 * {ece} 2.10 or later
+** Requires additional wildcard domains and certificates (which normally only
```

Comment:
@simitt @AlexP-Elastic Could you double check this?

Reply:
should this link to the ECE 2.10 docs which should have more details? I'm not sure it's actionable enough as it stands

Reply:
Yes, when the docs are available.